test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -36,6 +36,41 @@ public void elementsAreUnique() throws Exception {
         assertNotUnique(null, "a", "b", null);
     }
 
+    @Test
+    public void elementsHasAnyNull() throws Exception {
+
+        // empty list
+        assertHasNoNull();
```

Comment:
isAnyNull_noNull_returnsFalse()

Reply:
This should be used for the test name; not individual asserts!